SDL/Quintuplet.cu

```diff
@@ -843,19 +823,16 @@ __device__ bool SDL::passT5RZConstraint(struct SDL::modules& modulesInGPU, struc
         }
         if (i==2 || i==3){
             residual = (layeri <= 6 && ((side == SDL::Center) or (drdz < 1))) ? diffz : diffr;
-//            residual_missing=residual;
             float projection_missing=1;
         if (drdz<1)
             projection_missing = ((subdets == SDL::Endcap) or (side == SDL::Center)) ? 1.f : 1/sqrt(1+drdz*drdz); // cos(atan(drdz)), if dr/dz<1
         if (drdz>1)
             projection_missing = ((subdets == SDL::Endcap) or (side == SDL::Center)) ? 1.f : drdz/sqrt(1+drdz*drdz);//sin(atan(drdz)), if dr/dz>1
             error=error*projection_missing;
-            residual_missing=residual/error;
```

Comment:
I noticed that you removed this and I don't think it was part of my comments. Did you realize it wasn't needed at all or the removal was done by mistake?

Reply:
The residual_missing variable is defined for debugging and print out, so it's fine to be removed:)